addon/services/listen-analytics.js

```diff
@@ -197,63 +159,22 @@ export default Service.extend({
     let currentStreamName = get(currentStream, 'name');
 
     if (prevStreamName !== currentStreamName) {
```

Comment:
it's possible that this test is unnecessary due to checking if `currentSound !== previousStream` in `_onCurrentSoundChanged`